electrumsv/tests/test_wallet.py

```diff
@@ -151,9 +159,19 @@ def check_legacy_parent_of_standard_wallet(wallet: Wallet,
         # This account was created for the test and gets the correct flag
         assert account_keystore.get_masterkey_flags() == MasterKeyFlags.ELECTRUM_SEED
     else:
-        assert account_keystore.get_parent_keystore() is None
-        # These are pre-existing imported accounts and the information was never set and lost.
-        assert account_keystore.get_masterkey_flags() == MasterKeyFlags.NONE
+        original_wallet_migration_number = int(wallet.name()[0:2])
```

Comment:
This should not be using the version number. Look at the account and wallet. Identify the type of account and use that. The version number is only tangentially related and you are assuming it means something.

Reply:
I do not understand how this even makes sense, 27 gets conflicting checks anyway

Reply:
What am I supposed to use instead of the original migration number of the test wallet to distinguish between two StandardAccounts (one migrated from <27 and one migrated from >27)?

After they have been migrated to 29, they seem indistinguishable except that the older wallet's non-petty cash account (StandardAccount ) either will or wont have a parent keystore - but that is the thing we are checking for here so in the absence of some other marker... what else can be used?

Both have `keystore.BIP32_KeyStore` type
Both have `StandardAccount` type
Both have `DerivationType.BIP32` (edited)